src/config/config.rs

```diff
@@ -244,7 +243,11 @@ impl Field {
     directives
   }
   pub fn has_batched_resolver(&self) -> bool {
-    self.group_by.is_some()
+    if let Some(http) = &self.http {
+      !http.group_by.is_empty()
+    } else {
+      false
+    }
```

Comment:
```suggestion
    self.http.iter().all(|http| !http.group_by().is_empty())
```
May be this is a bit more concise. 
Can we add a quick unit test in the same file `config.rs` under `mod tests`. I think it will make sure we don't break this functionality later.

Reply:
Done. I finally used this instead:
```rust
self.http.as_ref().is_some_and(|http| !http.group_by.is_empty())
```